### PR TITLE
Update bigtable test support and start to add test for get_populations

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -7,8 +7,8 @@ require (
 	cloud.google.com/go/bigquery v1.0.1
 	cloud.google.com/go/bigtable v1.0.0
 	github.com/go-test/deep v1.0.3
-	github.com/golang/protobuf v1.3.4
-	github.com/google/go-cmp v0.3.1 // indirect
+	github.com/golang/protobuf v1.4.0-rc.4
```
Comment (hanlu09205) [inline go.mod:10]: For my education, when do we update those and where to find the right version?

File: store/core_test.go [hunk 2]
```diff
@@ -36,11 +36,11 @@ func TestGetPropertyLabels(t *testing.T) {
 	} {
 		jsonRaw, err := json.Marshal(d.labels)
 		if err != nil {
-			t.Fatalf("json.Marshal(%v) = %v", d.dcid, err)
+			t.Errorf("json.Marshal(%+v) = %+v", d.dcid, err)
```
Comment (hanlu09205) [inline store/core_test.go:39]: Nit, for err, I think %v is good enough.

File: store/populations_test.go [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+	"github.com/datacommonsorg/mixer/util"
+)
+
+func TestGetPopObs(t *testing.T) {
+	data := map[string]string{}
+	dcid := "geoId/06"
+	key := util.BtPopObsPrefix + dcid
+	btRow := `{
+	"name": "Santa Clara",
+  "populations": {
```
Comment (hanlu09205) [inline store/populations_test.go:31]: Nit, is there any tool we could use to format those?

File: store/populations_test.go [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+      "meanValue": 9.4461,
+      "measuredProp": "particulateMatter25",
+      "measurementMethod": "CDCHealthTracking",
+      "observationDate": "2014-03-20",
+      "observedNode": "geoId/06085"
+    }
+  ]
+}`
+
+	tableValue, err := util.ZipAndEncode(string(btRow))
```
Comment (hanlu09205) [inline store/populations_test.go:76]: So most of this is copying from g3 to github ? 

Lazy question, how to do run build and test on github?

File: go.mod (hunk 1 above)
Comment (shifucun) [reply go.mod:10]: This is automatically updated when I run "go test" locally. I think there is a way to force update this. Can check with Wei. In general, don't really need to update the package.

File: store/populations_test.go (hunk 3 above)
Comment (shifucun) [reply store/populations_test.go:31]: I use visual studio code and golang formatter plugin. It usually does good job. In this case as it is a string, so seems dumb. I updated manually.

File: go.mod (hunk 1 above)
Comment (hanlu09205) [reply go.mod:10]: Got it. Thanks

File: store/populations_test.go (hunk 4 above)
Comment (shifucun) [reply store/populations_test.go:76]: Yes. We setup a cloud trigger to run the test. If you scroll down and see there are "All checks have passed", the first one will run all the tests.

**Pull request merged**